trackmania.js

```diff
@@ -732,19 +811,19 @@ export function embedLeaderboardInfo(lb_info) {
         type: MessageComponentTypes.ACTION_ROW,
         components: [{
             type: MessageComponentTypes.STRING_SELECT,
-            custom_id: 'acc',
-            placeholder: 'Search player info',
+            custom_id: 'acc+lb',
+            placeholder: 'Search info of a Player',
             options: accounts,
         }],
     },{
         type: MessageComponentTypes.ACTION_ROW,
         components: [{
             type: MessageComponentTypes.STRING_SELECT,
-            custom_id: `lb+${timestamp}+p;${groupUid};${mapUid}`,
-            placeholder: 'Select page',
-            options: pages,
-        },],
-    },{
+            custom_id: 'acc+w',
```

Comment:
Are these `acc` params (`w` and `lb`) used? I'm probably just missing something

Reply:
no theyre not really used to mean anything, i just needed some sort of character to put there because discord yells at you if 2 components in a message have duplicate custom ids